js/classes.js

```diff
@@ -234,7 +236,8 @@ export class UserShip extends Ship {
         console.warn('Player is dead')
     }
     this.color = 'bg-purple'
-    const hit = new SoundEffect('lasers', '3.wav', 0.5, true).play()
+    const hit = new SoundEffect('lasers', '3.wav', 0.5, true);
```

Comment:
Maybe we can export all these in a soundEffects helper so that we know all the sounds we have available.